File: car/templates/CarTemplate.java
// name: car
import java.io.*;

public class CarTemplate {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

    static class TestCase {
        float v;
        float x;
    }
    
    public static TestCase read() throws java.lang.Exception {
        TestCase testCase = new TestCase();
        String[] info = br.readLine().split(":");
        testCase.v = Float.parseFloat(info[0]);
        testCase.x = Float.parseFloat(info[1]);
        return testCase;
    }

    public static void logic(TestCase testCase) {
        // v: A non-negative decimal value denoting the vehicle’s current speed (in m/s)
        // x: A positive decimal value denoting the obstacle distance from the front of the car (in m)
        float v = testCase.v;
        float x = testCase.x;

        // TODO: YOUR CODE HERE

        return;
    }

	public static void main(String[] args) throws java.lang.Exception {
        int T = Integer.parseInt(br.readLine());
        for (int i = 0; i < T; i++) {
            logic(read());
        }
    }
}

File: car/templates/car_template.py
name = "car"

def read():
    case = {}
    info = input().strip().split(":")
    case["v"] = float(info[0])
    case["x"] = float(info[1])
    return case

def logic(case):
    # v: A non-negative decimal value denoting the vehicle’s current speed (in m/s)
    # x: A positive decimal value denoting the obstacle distance from the front of the car (in m)
    v, x = case["v"], case["x"]
    
    # TODO: YOUR CODE

    if v == 0:
        return "SAFE"

    # using the basic formula to extract the time
    time = x / v

    if time <= 1:
        return "SWERVE"
    if time <= 5:
        return "BRAKE"
    else:
        return "SAFE"    

def main():
    T = int(input().strip())
    for _ in range(T):
        result = logic(read())
        print(result)

if __name__ == "__main__":
    main()